appveyor.yml

```diff
@@ -21,6 +21,6 @@ install:
   - bundle install --retry=3
 
 test_script:
-  - bundle exec rake
+  - bundle exec rake test
```

Comment:
I have to say I don't like this.. not obvious that `rake test` is a way to exclude rubinius